org.scala-ide.sdt.core/src/org/scalaide/ui/internal/editor/decorators/implicits/ImplicitHighlightingPresenter.scala

```diff
@@ -55,13 +55,12 @@ object ImplicitHighlightingPresenter {
       val txt = new String(sourceFile.content, t.pos.start, math.max(0, t.pos.end - t.pos.start)).trim()
       val pos = mkPosition(t.pos, txt)
       val region = new Region(pos.offset, pos.getLength)
-      val msg = compiler.asyncExec{
+      val msg = compiler.asyncExec {
         val sname = t.fun.symbol.nameString
         s"Implicit conversion found: `$txt`$DisplayStringSeparator`$sname($txt): ${t.tpe}`"
       }.getOption()
 
-      val annotation = new ImplicitConversionAnnotation(
-          () => compiler.mkHyperlink(t.symbol, name = "Open Implicit", region, scu.scalaProject.javaProject), msg.getOrElse(""))
+      val annotation = new ImplicitConversionAnnotation(t.pos, region, msg.getOrElse(""))
```

Comment:
so the main problem comes from the fact that `symbol` is quite heavy (I mean other objects can refer it too)?